src/pmpo_mesh.cpp

```diff
@@ -10,4 +10,25 @@ namespace polyMPO{
         return (geomType >mesh_unrecognized_lower &&
                 geomType <mesh_unrecognized_upper);
     }
+
+    void Mesh::setMeshVtxBasedFieldSize(int numVtxs){
+        PMT_ALWAYS_ASSERT(meshEdit_);
```

Comment:
Doesn't the mesh class know how many vertices it has?  Why pass it in as an argument... seems like it is inviting problems.

Reply:
Good catch, here is the commit: https://github.com/SCOREC/polyMPO/commit/395733aa690fc199e6af8cbe9c53a057e56c9c98